action.yml

```diff
@@ -12,10 +12,13 @@ inputs:
   github-token:
     description: 'A GitHub auth token to be able to create the pull request'
     required: true
-  gradle-path:
+  version-filepath:
```

Comment:
I would leave this as gradle-path, as it better describes what its relating to, the use of "version" is not particularly clear, also this could be a breaking change where as all the other changes are not breaking, and its specific to gradle, whereas we also deal with pom's in this script but we don't use the variable for poms

Reply:
Yeah. well; I did not find where we were using this gradle-path to begin with. So having it and using it instead of having it and not using it is 100% an improvement.

This being said, there is no good reason to:

1. rename it to `version-filepath` (and lose the "coupling" to Gradle) - so far so good...
2. add it to the Gradle part of the code - yep, great, thank you..
3. *not* do it for the Maven part...

I mean supporting the Maven part of this is not the most fun, but it's out there, and if you go down the route of un-specialising those variable/property names, that's fine, but if you do, you really need to do the Maven part.

If not, then keep it `gradle-path` and let us be happy with the improvement on the Gradle part of the tool.